main.js

```diff
@@ -5,28 +5,51 @@ const authorInput = document.getElementById('author');
 
 let bookss = [];
 
-const storedBooks = JSON.parse(localStorage.getItem('bookss'));
+class BookClass {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  bookCode() {
+    return `<li>
+    <p>${this.title}</p>
+    <p>${this.author}</p>
+    <button type="button" data-id='${this.id}' class="remove">remove</button>
+    <hr>
+  </li>`;
+  }
 
-function singleBook(book) {
-  return `<li>
-      <p>${book.title}</p>
-      <p>${book.author}</p>
-      <button type="button" data-id='${book.id}' class="remove">remove</button>
-      <hr>
-    </li>`;
+  static addBook(book) {
+    let id = 1;
+    if (bookss.length > 0) {
+      id = bookss[bookss.length - 1].id + 1;
+    }
+    book.id = id;
+    bookss.push(book);
+    localStorage.setItem('bookss', JSON.stringify(bookss));
+  }
+
+  static remove(id) {
+    bookss = bookss.filter((b) => b.id !== Number(id));
+    localStorage.setItem('bookss', JSON.stringify(bookss));
+  }
 }
 
+const storedBooks = JSON.parse(localStorage.getItem('bookss'));
+
 function showBooks() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
@rloterh Thank you 💯